clients/banking/src/components/TransferInternationalWizardBeneficiary.tsx

```diff
@@ -147,16 +160,37 @@ export const TransferInternationalWizardBeneficiary = ({
     listenFields(["results"], () => refresh());
   }, [listenFields, refresh]);
 
+  useEffect(() => {
+    if (isNotNullish(errors) && errors.length) {
+      for (const error of errors) {
+        if (isNotNullishOrEmpty(error[0])) {
+          dynamicFormApiRef.current?.setDynamicFieldError(error[0], error[1]);
+        }
+      }
+    }
+  }, [setFieldError, errors, dynamicFormApiRef.current?.setDynamicFieldError]);
+
+  const orphanErrors = useMemo(() => errors?.filter(([key]) => isNullish(key)), [errors])?.map(
+    ([_, message]) => message,
+  );
+
   return (
     <View>
-      <Tile>
-        {errors?.map((message, i) => (
-          <View key={`validation-alert-${i}`}>
-            <LakeAlert variant="error" title={message} />
-            <Space height={12} />
-          </View>
-        ))}
-
+      <Tile
+        footer={
+          isNotNullish(orphanErrors) && orphanErrors.length ? (
```

Comment:
Avoid non-boolean values in conditions (`orphanErrors.length` -> `orphanErrors.length > 0`).
It often create issues, especially in react renders.

For example:

```ts
{orphanErrors.length && <p>Hello</p>} // will print the text "0" is array is empty
```